quantized_mesh_encoder/encode.py

```diff
@@ -271,3 +275,14 @@ def write_edge_indices(f, positions, n_vertices):
 
     f.write(pack(NP_STRUCT_TYPES[np.uint32], len(top)))
     f.write(top.tobytes())
+
+def write_vertex_normals(f, positions, indices):
+    normals = compute_vertex_normals(positions, indices)
+    encoded = oct_encode(normals)
+    encoded = encoded.flatten('C')
+    encoded = encoded.tobytes('C')
+
+    s_normals = len(encoded)
+    f.write(pack(EXTENSION_HEADER['extensionId'], 1))  # octvertexnormals extension is 1 in the spec
```

Comment:
Might be nice to store this `1` either as a top-level constant variable or create an enum with all the extension ids